### PR TITLE
Boom

### PR DESCRIPTION
**Pull request by stkem**
@andrewconner @leogrim @eishay 
quick review?

### CODE REVIEW

**Comment by stkem** [comment]
Will follow up with a blog post on this Tuesday.

**Comment by eishay** [comment]
thanks!

File: README.md [hunk 1]
```diff
@@ -1 +1,87 @@
-# ReactiveLock
+Reactive programming and locking patterns don't go together? Think again. ReactiveLock, as the name might suggest, is a Reactive Lock. I.e. it is a construct that allows you to employ patterns similar to classic thread/lock concurrency without actually blocking any threads, thus making it possible to use efficient small thread pools.
+
+It's great for transitioning a codebase from a classic threads & locks pattern to a more reactive environment and it's also great for transitioning a skill set from those classic patterns.
+
+Even with perfectly thread safe code a `ReactiveLock` can come in really handy to simply throttle a piece of code. There are many scenarios where reducing concurrency can actually increase throughput, e.g. when cometing for IO resources.
```
Comment (andrewconner) [inline README.md:5]: s/cometing/competing/

Comment (stkem) [inline README.md:5]: oops, fixing.

**Pull request merged**